### ISSUE TITLE
Future Update - Incorporate Crypto Tracker

### ISSUE THREAD

**Issue by shadowerwtas** [daisy613/LHPControl#3]
As a future update it would be nice a crypto tracker to be incorporated in LHPC. Either embedded or to run externally as it does via a web browser now.

One control panel, to rule them all :P

**Comment by daisy613** [comment]
I agree. In fact, I am looking into converting the current WPF-based LHPControl into a python app running in the background as a local site. That way, we can add a local DB that collects stats and allows for a nice pretty tracker to be integrated :)
